etc/onespan-sign-node.api.md

```diff
@@ -146,7 +217,43 @@ interface CreatePackage {
 }
 
 // @public
-export type CreatePackageData = PackageData;
+type CreatePackageData = PackageData;
+
+// @public (undocumented)
+type CreateSenderData = RecursivePartial<Sender>;
+
+// @public
+export type CustomData = Nullable<Record<string, any>>;
```

Comment:
Just out of curious – what type of data would go into `CustomData`?

Reply:
It's user-defined key/values that can be passed into the object. For example, we could pass in an `envId` value into this custom data, and it'll be preserved and returned when we fetch the entity from OneSpan